src/main/kotlin/main.kt

```diff
@@ -9,17 +9,16 @@ fun main() {
 
     val parser = RequestParser()
 
-    val badRequestController = BadRequestController()
-    val notFoundController = NotFoundController()
-    val router = Router(badRequestController, notFoundController)
+    val router = Router()
     router.addRoute("GET", "/simple_get", SimpleGetController())
     router.addRoute("GET", "/simple_get_with_body", SimpleGetWithBodyController())
     router.addRoute("HEAD", "/simple_get", SimpleGetController())
```

Comment:
@skp96 noticing that there _is_ a route that uses two different methods.

Reply:
The endpoint `/simple_get` responds to two different methods per the acceptance tests for simple get and simple head feature